### DISCUSSION TITLE
How about releasing a build of the updated new version?

### DISCUSSION THREAD

**Discussion by cplus98** [citizenll/godot-flexbox#16] [General]
A new build is needed to resolve issues running on iOS and Mac. :)

**Comment by citizenll** [comment]
Bro, just hold on for a bit, I'm gonna buy a Mac Mini first.

**Comment by citizenll** [comment]
> A new build is needed to resolve issues running on iOS and Mac. :)

Add my Discord account, bro. That way we can quickly and conveniently communicate about some issues.
`
citizenl1024
`
